config/initializers/opentelemetry.rb

```diff
@@ -54,6 +54,9 @@
       'OpenTelemetry::Instrumentation::Sidekiq' => {
         span_naming: :job_class, # Use the job class as the span name, otherwise this is the queue name and not very helpful
       },
+      'OpenTelemetry::Instrumentation::Redis' => {
+        trace_root_spans: false, # don't start traces with Redis spans
```

Comment:
Maybe an instance operator _wants_ to see all the Redis activity? I have not yet figured out a way to make these options more convenient for operators to customize without them overwriting this initializer file somehow.